kubernetes/windows/hostlogs/main.ps1

```diff
@@ -86,13 +86,19 @@ Set-EnvironmentVariables
 Start-FileSystemWatcher
 
 if(Get-GenevaEnabled){
```

Comment:
Get-GenevaEnabled isn't so much checking for enablement, but for a valid and complete configuration. We should likely include an output which describes why this check failed so users can easily see "oh I need to set X"

Reply:
Same terms used by CI: https://github.com/microsoft/Docker-Provider/blob/cae75f9854df1842b07886a7bbb3a5ee065945eb/kubernetes/windows/hostlogs/main.ps1#L67

Reply:
Not sure there is a benefit from that as the customer onboarding documenation will state this.

Reply:
> Same process used by CI:
> 
> https://github.com/microsoft/Docker-Provider/blob/cae75f9854df1842b07886a7bbb3a5ee065945eb/kubernetes/windows/hostlogs/main.ps1#L67

That's our file. 
CI sets the GENEVA_LOGS_INTEGRATION env var in the parser – https://github.com/microsoft/Docker-Provider/blob/901bbb0895c81fccbcfa68bafad2ee366dafdd4a/build/common/installer/scripts/tomlparser-geneva-config.rb#L275
Then they check that var – https://github.com/microsoft/Docker-Provider/blob/901bbb0895c81fccbcfa68bafad2ee366dafdd4a/kubernetes/windows/main.ps1#L739

I think Get-GenevaConfigured would be a better name for this. Or we could switch to doing it in the parser to match CI

Reply:
Moved env vars and renamed this check